test/regression/2015-08-05-symbolic-const-variable.c

```diff
@@ -0,0 +1,17 @@
+// RUN: %llvmgcc %s -emit-llvm -g -O0 -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out -exit-on-error -exit-on-error-type=user %t.bc 2> %t.log
```

Comment:
Your test is broken :)   It will alway pass, because KLEE exits with the error that `-exit-on-error-type=user` is not a valid option.    Also, why do you use both `-exit-on-error` and `-exit-on-error-type`?